google/cloud/bigquery/big_query_read_options.gcpcxx.pb.h

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated by the Codegen C++ plugin.
+// If you make any local changes, they will be lost.
+// source: google/cloud/bigquery/storage/v1/storage.proto
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGQUERY_BIG_QUERY_READ_OPTIONS_GCPCXX_PB_H
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGQUERY_BIG_QUERY_READ_OPTIONS_GCPCXX_PB_H
+
+#include "google/cloud/bigquery/big_query_read_connection.gcpcxx.pb.h"
+#include "google/cloud/bigquery/big_query_read_connection_idempotency_policy.gcpcxx.pb.h"
+#include "google/cloud/backoff_policy.h"
+#include "google/cloud/options.h"
+#include "google/cloud/version.h"
+#include <memory>
+
+namespace google {
+namespace cloud {
+namespace bigquery {
+inline namespace GOOGLE_CLOUD_CPP_GENERATED_NS {
+
+struct BigQueryReadRetryPolicyOption {
```

Comment:
Would it be possible to inject a generic comment here like "/// An option to use with `google::cloud::Options`"? Some tie back go `google::cloud::Options` may be helpful for customers.

A followup PR is fine.